docs/how-to/rocm-for-ai/index.rst

```diff
@@ -6,11 +6,11 @@
 Using ROCm for AI
 *****************
 
-ROCm offers a suite of optimizations for AI workloads from large language models (LLMs) to image and video detection and
+ROCm™ offers a suite of optimizations for AI workloads from large language models (LLMs) to image and video detection and
```

Comment:
```suggestion
ROCm offers a suite of optimizations for AI workloads from large language models (LLMs) to image and video detection and
```
TM is only required once in a page. If you look at the top left of the rendered webpage, every page has TM included by default.

Reply:
It was done by design so I could be lazy on this. You might as well take advantage.

Reply:
Ah, I didn't realize that. Thanks @saadrahim.